config/packages/csrf.php

```diff
@@ -6,7 +6,7 @@
 use Symfony\Component\HttpKernel\Kernel;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
-    // This configuration file is specific to Symfony 7.2+
+    // This configuration file is specific to Symfony 7.3+
```

Comment:
This doesn't match with the line below? `if (Kernel::VERSION_ID < 70200) {`

Reply:
Indeed. In fact, it should be “Symfony 7.1 and above”.